BitioExample.java

```diff
@@ -8,9 +8,9 @@
 public class BitioExample {
    public static void main(String args[]) {
       Connection c = null;
-      String bitApiKey = "<your api key>";
-      String bitDB = "sensors";
-      String bitUser = "adam";
+      String bitApiKey = "<your_bitdotio_password>";
```

Comment:
This is not your actual password - it's still the connect key from the connect dialog.